microprofile/lra/jax-rs/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<beans xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://xmlns.jcp.org/xml/ns/javaee"
```

Comment:
bean-discover-mode=all is not allowed within Helidon.
Also as there is an extension, it may be better to add all required beans in the extension (and remove bean defining annotations from them).
`beans.xml` should only be in examples and archives that do not contain an extension.
This is to prevent as many problems as possible when somebody uses shading.

Reply:
make sense sorry, removed all annotations and bean.xml, beans are registered by cdi extension now